core/strings/strings.odin

```diff
@@ -1464,12 +1483,24 @@ Output:
 
 */
 last_index_byte :: proc(s: string, c: byte) -> (res: int) {
-	for i := len(s)-1; i >= 0; i -= 1 {
-		if s[i] == c {
-			return i
+	_last_index_byte :: #force_inline proc(s: string, c: byte) -> int {
+		for i := len(s)-1; i >= 0; i -= 1 {
```

Comment:
'

Reply:
See comment above about `for x in y` iteration over strings.